RHIDP-3393 - Document how users can access Plugin Marketplace in RHDH (#995)
redhat-developer/red-hat-developers-documentation-rhdh: assemblies/dynamic-plugins/assembly-extensions-plugins.adoc

[id="rhdh-extensions-plugins_{context}"]

> = Extensions in {product}

Reviewer: Since this is a TP feature, should we include an Important note in this section?
For example: 
![image](https://github.com/user-attachments/assets/bb85563c-3f5e-4a0f-92eb-403efd56ad2c)

Author: @hmanwani-rh Yes, added.

:context: rhdh-extensions-plugins

> {product} ({product-very-short}) includes the Extensions feature which is preinstalled and enabled by default. Extensions provides users with a centralized interface to browse and manage available plugins 

Reviewer 2: does "Extensions Catalog" work better in docs? Maybe ask CF?

Reviewer 2: /lgtm hard to read in this format, but OK I think. I would prefer a link to a rendering in future if possible. Thanks @Gerry-Forde

Author: Thanks for the review @benwilcock - I did consider using the term "Extensions Catalog" but @christophe-f advised against using it as it might cause confusion with the Catalog feature itself.

BTW - there is a link to the [GitHub preview](https://redhat-developer.github.io/red-hat-developers-documentation-rhdh/pr-995/plugins-rhdh-install/#rhdh-extensions-plugins_assembly-install-third-party-plugins-rhdh) in the PR description above.

[id="rhdh-extensions-plugins_{context}"]
= Extensions in {product}

include::{docdir}/artifacts/snip-technology-preview.adoc[]

:context: rhdh-extensions-plugins

{product} ({product-very-short}) includes the Extensions feature which is preinstalled and enabled by default. Extensions provides users with a centralized interface to browse and manage available plugins 

> Your organization can use Extensions to discover plugins that extend {product-very-short} functionality, streamline development workflows, and improve the developer experience.

Reviewer: ```suggestion
You can use Extensions to discover plugins that extend {product-very-short} functionality, streamline development workflows, and improve the developer experience.
```
